src/Bicep.Core/TypeSystem/Radius/V3/KnownComponents.cs

```diff
@@ -81,6 +81,68 @@ public static ComponentData MakeContainer()
                 functions: null);
             var portsProperty = new TypeProperty("ports", portsType, TypePropertyFlags.None);
 
+            var headersType = new ObjectType(
+                name: "headers",
+                validationFlags: TypeSymbolValidationFlags.Default,
+                properties: Array.Empty<TypeProperty>(),
+                additionalPropertiesType: LanguageConstants.String,
+                additionalPropertiesFlags: TypePropertyFlags.None,
+                functions: null);
+
+            var httpGet = new ObjectType(
+                name: "httpGet",
+                validationFlags: TypeSymbolValidationFlags.Default,
+                properties: new TypeProperty[] {
+                    new TypeProperty("containerPort", LanguageConstants.Int, TypePropertyFlags.Required),
+                    new TypeProperty("path", LanguageConstants.String, TypePropertyFlags.Required),
+                    new TypeProperty("headers", headersType, TypePropertyFlags.None),
+                    new TypeProperty("kind", new StringLiteralType("httpGet"), TypePropertyFlags.Required),
+                    new TypeProperty("initialDelaySeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("failureThreshold", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("periodSeconds", LanguageConstants.Int, TypePropertyFlags.None),
```

Comment:
Can you add descriptions for these things? They show up in the editor when you're typing. It's fine to copy-paste from the schema since you already wrote them once.